server/runner/backuprun/backup_runner.go

```diff
@@ -1,4 +1,5 @@
-package server
+// Package backuprun is the runner for backups.
+package backuprun
```

Comment:
"backuprun" sounds strange.

Reply:
Yeah, couldn't think of a better name. See other comment. Let's use this for now, and we change it later.